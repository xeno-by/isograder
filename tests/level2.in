* Section 1 [/1]
** Subsection 1 [/1]
*** Subsubsection 1 [/1]
** Subsection 2 [/0]
