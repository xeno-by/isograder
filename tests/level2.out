EXIT CODE:
0

STDOUT:
* Section 1 [1/1]
** Subsection 1 [1/1]
*** Subsubsection 1 [1/1]
** Subsection 2 [0/0]

STDERR:

